catalog/transformer/catalog-transformer-mgmp/pom.xml

```diff
@@ -0,0 +1,143 @@
+    <name>Alliance :: Catalog :: Transformer :: MGMP</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>xml-apis</groupId>
+            <artifactId>xml-apis</artifactId>
+            <version>1.4.01</version>
+        </dependency>
+        <dependency>
+            <groupId>ddf.catalog.core</groupId>
+            <artifactId>catalog-core-api</artifactId>
+            <version>${ddf.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>ddf.catalog.core</groupId>
+            <artifactId>catalog-core-api-impl</artifactId>
+            <version>${ddf.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.codice.ddf.spatial</groupId>
+            <artifactId>spatial-csw-transformer</artifactId>
+            <version>${ddf.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.codice.alliance.catalog.core</groupId>
+            <artifactId>catalog-core-api-impl</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.codice.alliance.catalog.core</groupId>
+            <artifactId>catalog-core-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>ddf.platform.util</groupId>
+            <artifactId>platform-util</artifactId>
+            <version>${ddf.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.codice.ddf.spatial</groupId>
+            <artifactId>spatial-csw-common</artifactId>
+            <version>${ddf.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Export-Package>
+                            org.codice.alliance.catalog.core.api.types,
+                            org.codice.ddf.spatial.ogc.csw.catalog.transformer,
+                            org.codice.ddf.spatial.ogc.csw.catalog.converter,
+                            org.codice.ddf.spatial.ogc.catalog.common.converter,
+                            org.codice.ddf.spatial.ogc.csw.catalog.common,
+                            org.codice.ddf.spatial.ogc.csw.catalog.common.converter,
+                            org.codice.ddf.spatial.ogc.csw.catalog.common.transaction,
+                            org.codice.ddf.spatial.ogc.csw.catalog.common.transformer
+                        </Export-Package>
+                        <Embed-Dependency>
+                            platform-util,
+                            catalog-core-api-impl
```

Comment:
It does get them both, but I prefer the more explicit notation when there's a name collision.